openshift/Makefile

```diff
@@ -14,3 +14,6 @@ manifests: $(KUSTOMIZE) $(YQ)
 .PHONY: verify-manifests
 verify-manifests: manifests
 	git diff --exit-code
+
+.PHONY: test-e2e
+test-e2e: /bin/true ## Run the e2e tests. TODO: stub until tests are working downstream
```

Comment:
The `/bin/true` shouyld be part of the recipe, indented on a separate line.

Reply:
Woops!